python/ray/worker.py

```diff
@@ -1682,16 +1682,27 @@ def kill(actor):
 
 
 def cancel(object_id, force=False):
-    """Kill a task forcefully.
+    """Cancels a locally-submitted task according to the following pseudocode.
 
-    This will interrupt any running tasks on the actor, causing them to fail
-    immediately. Any atexit handlers installed in the actor will still be run.
+    If the specified task is pending execution, it will not be executed. If
+    the task is currently executing, the behavior depends on the `force` flag.
```

Comment:
sphinx requires double backticks for monospace text. so
`force` -> ``force`` and same with the following code snipeets

Reply:
opps github eat the backticks. I was trying to say:
```
instead of `force`
do ``force``
```

Reply:
Oh yikes, got it!